tutorials/kernelshap_tabular_penguin.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Ah! I found something! You can use your own DIANNA vis tool here!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Well spot! Thanks. It is updated now.</p>